Keas.Jobs.SendMail/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿
+using System;
+using System.IO;
+using System.Linq;
+using Keas.Core.Data;
+using Keas.Core.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+using Keas.Core.Services;
+
+namespace Keas.Jobs.SendMail
+{
+    public class Program
+    {
+        public static IConfigurationRoot Configuration { get; set; }
+        public static IServiceProvider Provider { get; set; }
+
+        public static IEmailService EmailService { get; set; }
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+
+
+            // Use this to get configuration info, environmental comes in from azure
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json");
+            var environmentName = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+            if (string.Equals(environmentName, "development", StringComparison.OrdinalIgnoreCase))
+            {
+                builder.AddUserSecrets<Program>();

```

Comment:
@srkirkland I don't know if there is a better/updated way to do this, but this is how I did it with anlab

Reply:
@jSylvestre yes that is old and you should not need that anymore